http/router.go

```diff
@@ -121,6 +121,25 @@ func (p *AddressTxsConfirmedPayload) ToNotification(query *MobilePushWebHookQuer
 	}
 }
 
+type SwapUpdatedPayload struct {
+	Event string `json:"event" binding:"required,eq=swap.update"`
```

Comment:
Is this supposed to be `swap_updated`?
```suggestion
	Event string `json:"event" binding:"required,eq= swap_updated"`
```

Reply:
No it's the JSON structure as specified by the Boltz webhook: https://docs.boltz.exchange/v/api/webhooks